pysnooper/tracer.py

```diff
@@ -20,6 +20,7 @@
 
 
 ipython_filename_pattern = re.compile('^<ipython-input-([0-9]+)-.*>$')
+ansible_filename_pattern = re.compile('^(/.+zip)/(ansible/modules/.+)$')
```

Comment:
Please use a raw string here, and also fix the line before it to be a raw string. 

I want to be extra careful in the regex so we won't have any false positives. Can you write a test specifically for the regex pattern? It should have around 10 different paths, some positives and some negatives, and ensure the regex is accurate for all of them.

Reply:
- Regex changed to get a better match and changed to raw string
- Tests added to test the regex with different cases